Build hangs

In PR #56 the build hangs after what seemingly looks like a green test suite, needs to be fixed so we can merge the PR.

Actually there are two problems here. The jdbc connector is also unable to connect to the db in #56.

```code
Fetch Size limited to: 100000 rows
Fetch max data chunk size limited to: 300
Server started!
Connecting to database...
Connecting to database...
com.mysql.cj.jdbc.exceptions.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
        at com.mysql.cj.jdbc.exceptions.SQLError.createCommunicationsException(SQLError.java:174)
        at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:64)
        at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:832)
        at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:456)
        at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:240)
        at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:207)
        at java.sql.DriverManager.getConnection(DriverManager.java:664)
        at java.sql.DriverManager.getConnection(DriverManager.java:247)
        at qlik.jdbc.connector.ConnectorImpl.getData(ConnectorImpl.java:62)
        at qlik.connect.ConnectorGrpc$MethodHandlers.invoke(ConnectorGrpc.java:298)
        at io.grpc.stub.ServerCalls$UnaryServerCallHandler$UnaryServerCallListener.onHalfClose(ServerCalls.java:171)
        at io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.halfClosed(ServerCallImpl.java:283)
        at io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1HalfClosed.runInContext(ServerImpl.java:707)
        at io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
        at io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
Caused by: com.mysql.cj.exceptions.CJCommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
        at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
        at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:105)
        at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:151)
        at com.mysql.cj.exceptions.ExceptionFactory.createCommunicationsException(ExceptionFactory.java:167)
        at com.mysql.cj.protocol.a.NativeSocketConnection.connect(NativeSocketConnection.java:91)
        at com.mysql.cj.NativeSession.connect(NativeSession.java:152)
        at com.mysql.cj.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:952)
        at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:822)
        ... 15 more
Caused by: java.net.UnknownHostException: mysql-database: Name does not resolve
        at java.net.Inet4AddressImpl.lookupAllHostAddr(Native Method)
        at java.net.InetAddress$2.lookupAllHostAddr(InetAddress.java:928)
        at java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1323)
        at java.net.InetAddress.getAllByName0(InetAddress.java:1276)
        at java.net.InetAddress.getAllByName(InetAddress.java:1192)
        at java.net.InetAddress.getAllByName(InetAddress.java:1126)
        at com.mysql.cj.protocol.StandardSocketFactory.connect(StandardSocketFactory.java:150)
        at com.mysql.cj.protocol.a.NativeSocketConnection.connect(NativeSocketConnection.java:65)
        ... 18 more
```

--- a/example/reload-runner/index.js
+++ b/example/reload-runner/index.js
@@ -87,7 +87,7 @@ async function load() {
   try {
     await loadData(app, postgresqlConnectionSettings);
   } catch (e) {
-    console.log(`Failed to load data from PostgresSQL with error: ${e}`);
+    console.log(`Failed to load data from PostgreSQL with error: ${e}`);
     process.exit(1);
   }
 

--- a/example/reload-runner/index.js
+++ b/example/reload-runner/index.js
@@ -76,10 +76,20 @@ async function load() {
   }
 
   console.log('Loading from MySQL');
-  await loadData(app, mysqlConnectionSettings);
+  try {
+    await loadData(app, mysqlConnectionSettings);
+  } catch (e) {
+    console.log(`Failed to load data from MySQL with error: ${e}`);
+    process.exit(1);
+  }
 
   console.log('Loading from PostgreSQL');
-  await loadData(app, postgresqlConnectionSettings);
+  try {
+    await loadData(app, postgresqlConnectionSettings);
+  } catch (e) {
+    console.log(`Failed to load data from PostgresSQL with error: ${e}`);
+    process.exit(1);
+  }
 
   await global.deleteApp(appId);
 

